addons/io/org.openhab.io.homekit/README.md

```diff
@@ -60,6 +60,12 @@ A full list of supported accessory types can be found in the table below.
   <td>An accessory that provides a single read-only value indicating the relative humidity.</td>
  </tr>
  <tr>
+  <td>Lock</td>
```

Comment:
The current tags are all part of the emerging ESH ontology, and are meant to be consistent across io add-ons (i.e. The Amazon Echo add-on). There's not yet one defined for Lock. 

There's an ongoing discussion in https://github.com/eclipse/smarthome/issues/1093 about extending the ontology. I think Lock is a good example of where we need to provide a mechanism for enum values. Using a Switch is okay as a fallback, but it can't capture the full possibilities of a Lock state - for example, HomeKit can accept a "Jammed" state.

In order to make sure we're being consistent, I think it would be a good idea to hold off on the Lock support until we get the ontology question settled. We don't really want users adding these tags, only to have us break them later.